script/Arcueid/Day 8/08_00_ARC08_2.txt

```diff
@@ -146,7 +146,7 @@ Exactly who is that woman to you, Nii-san?"
 [sha:fcd3d0c77df9de8f86ba4b143a1922c4d9dce15d]{
 -- Page 65, Offset 19838.
 -- 「ふうん。夜中に出かけるほどのご友人なんですね。
-"Hmm. Rather close friends, for the two of you to be going out at night like that.
+"Hmm. Rather close friends for the two of you to be going out at night like that.
```

Comment:
I kinda like the comma there, but also maybe
> Rather %{i}close%{/i} friends

Reply:
This comma is pretty silted to me no matter who i read it aloud. I'd rather it stay gone.

I also prefer the italics below